app/views/items/_items.html.erb

```diff
@@ -0,0 +1,3 @@
+<% items.each do |item| %>
```

Comment:
I would still suggest you make this partial represent a single item, instead of looping through inside the partial.